automation/jleechanorg_pr_automation/automation_safety_manager.py

```
@@ -711,10 +711,8 @@ def main():
     """CLI interface for safety manager"""
 
     parser = argparse.ArgumentParser(description="Automation Safety Manager")
-    parser.add_argument(
-        "--data-dir",
-        default=os.path.join(os.path.expanduser("~"), ".automation_safety"),
-        help="Directory for safety data files")
+    parser.add_argument("--data-dir", default="/tmp/automation_safety",
```

Ruff (0.14.7)

Probable insecure usage of temporary file or directory: "/tmp/automation_safety"

(S108)